posts/2021-03-13-microprofile-config-2.0.adoc

```diff
@@ -31,7 +31,7 @@ Then, it's just 2 steps to launch:
 
 1. You’ll need to tell your build where all the awesome new features are defined. 
```

Comment:
Change,
"You’ll need to tell your build where all the awesome new features are defined."
to,
"You need to tell your build where all the awesome new features are defined."